templates/municipal_chief/adherent/list.html.twig

```diff
@@ -0,0 +1,64 @@
+{% extends 'municipal_chief/_layout.html.twig' %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    <link href="{{ asset('select2/select2.min.css') }}" rel="stylesheet">
+{% endblock %}
+
+{% block municipal_chief_content %}
+    <div class="datagrid__pre-table b__nudge--bottom-larger">
+        <h3>Adhérents</h3>
+        <span class="datagrid__count">
+            <span>{{ results.count |number_format(0, ',', ' ') }}</span> sur {{ results.totalItems |number_format(0, ',', ' ') }} adhérent{{ total_count > 1 ? 's' : '' }}
```

Comment:
Can you glue the filters twig please ? 
`{{ results.count |number_format ..}}` => `{{ results.count|number_format ... }}`